sites/default/statement.inc.php

```diff
@@ -1,6 +1,7 @@
 <?php
```

Comment:
Actually, disregard above comment, since each site will accept different cards and have different images.